include/inviwo/core/properties/isotfproperty.h

```diff
@@ -75,6 +75,10 @@ class IVW_CORE_API IsoTFProperty : public CompositeProperty,
 
     virtual std::string getClassIdentifierForWidget() const override;
 
+    virtual void set(const Property* property) override;
+    void set(const IsoValueProperty& p);
+    void set(const TransferFunctionProperty& p);
```

Comment:
Would be good with documentation on what happens to iso values. Are they removed?